### PR TITLE
[CAS-431] Add resource prefix Gradle config, make existing resources comply

### PR DESCRIPTION
**Pull request by zsmb13**
https://stream-io.atlassian.net/browse/CAS-431

### Description

Adds Gradle configuration for prefixing resources with `stream_` in modules where we have resources. Updates all current resources (except for styles and styleable declarations) to comply with this rule. Removes some unused resources.

Resources that don't have a prefix now result in lint errors:

image

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] ~Changelog updated with client-facing changes~
- [ ] ~New code is covered by unit tests~
- [ ] ~Comparison screenshots added for visual changes~
- [x] Reviewers added

### CODE REVIEW

**Review by samiuelson — APPROVED** [review]
Nice improvement 👍🏼

File: stream-chat-android-client/build.gradle [hunk 1]
```diff
@@ -35,6 +35,9 @@ android {
             consumerProguardFiles 'consumer-proguard-rules.pro'
         }
     }
+
+    resourcePrefix 'stream_'
```
Comment (JcMinarro) [inline stream-chat-android-client/build.gradle:39]: I didn't know it was integrated on the Gradle Plugin, I though it was a Lint Rule we needed to write.
Cool!!!

Comment (JcMinarro) [inline stream-chat-android-client/build.gradle:39]: I didn't know it was integrated on the Gradle Plugin, I though it was a Lint Rule we needed to write.
Cool!!!

**Review by JcMinarro — COMMENTED** [review]
As far as all our Modules are "Android Modules", shouldn't we add this rule to all of them? Maybe we don't have resources for now, but because they are Android Modules they could have

**Comment by zsmb13** [comment]
Yeah, I skipped the modules that don't have any resources in them, for now.

**Pull request merged**